pom.xml

```diff
@@ -87,7 +87,17 @@
 			<artifactId>jakarta.ws.rs-api</artifactId>
 			<version>2.1.6</version>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
JUnit was already in the dependancies and did not need to be added

Reply:
adapted